### DISCUSSION TITLE
MenuHorizontal.Razor

### DISCUSSION THREAD

**Discussion by mikecasas** [oqtane/oqtane.framework#687] [General] [answered]
Hi @sbwalker, this may be a trivial request, but I noticed in the repo that the razor file Oqtane.Client/Themes/Controls/MenuHorizontal has a capital 'R' for the razor file extension name. When I do a build in Docker it throws a warning. When I change the file extension to lower *.razor it works.

I tried doing a PR, but just changing the file extension case did not record a change in git. Maybe my git settings are case insensitive. Could you rename the file to MenuHorizontal.razor, or tell me how to change my settings to pick up the file change, so I can submit the PR?

Thanks!

**Comment by jimspillane** [comment, marked as answer]
@mikecasas did you try [renaming](https://docs.github.com/en/github/managing-files-in-a-repository/renaming-a-file) the file in GitHub?

![image](https://user-images.githubusercontent.com/15007083/90518054-3f040b00-e134-11ea-8699-18e553c43ac1.png)

**Reply by mikecasas** [reply]
Thanks, yes I was able to make the change and submit the PR. Great suggestion.

**Reply by mikecasas** [reply]
PR #688
